src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_de.properties

```diff
@@ -3,6 +3,7 @@ assignment.inner.avoid=Innere Zuweisungen sollten vermieden werden.
 avoid.clone.method=Die Methode clone() sollte vermieden werden.
 avoid.double.brace.init=Vermeiden Sie doppelte geschweifte Klammern.
 avoid.finalizer.method=Die Methode finalize() sollte vermieden werden.
+constructors.declaration.grouping=Alle Konstruktoren sollten zusammengefasst werden. Der vorherige Konstruktor befand sich in der Zeilennummer „{0}“.
```

Comment:
@Zopsss , looks like you have been copy-pasting entire line into the translator?
The quotes are also been styled i18n for few files.. Please look into such

Reply:
Fixed. thanks for pointing this out.